angel.py

```diff
@@ -27,10 +27,16 @@
 import random
 import logging
 import datetime
-# # FROMS
+import yaml
+import os
+
+# FROMS
 from models import Player
 from arrange import angel_mortal_arrange
 
+# Instantiate Logger
```

Comment:
Could you please implement this fix in a standalone branch/PR! No need to remove it from this branch tho

Reply:
This section of code seems to be duplicated. Could we abstract a logger script that does this? The singleton pattern may be applicable for this usage (so multiple scripts reference the same logger object)